redundancy/interface.go

```diff
@@ -0,0 +1,20 @@
+package redundancy
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+// NodeRedundancyHandler provides functionality to handle the redundancy mechanism for a node
+type NodeRedundancyHandler interface {
```

Comment:
Why do we have the interface here? Should have been wherever we consume the interface. You might redefine-it in all packages used (consensus, node....)

Reply:
Yep, will move there